core/lib/zksync_core/src/api_server/web3/tests/snapshots.rs

```diff
@@ -0,0 +1,125 @@
+//! Tests for the `snapshots` Web3 namespace.
+
+use std::collections::HashSet;
+
+use zksync_types::block::{BlockGasCount, L1BatchHeader};
+use zksync_web3_decl::namespaces::SnapshotsNamespaceClient;
+
+use super::*;
+use crate::state_keeper::tests::create_l1_batch_metadata;
+
+async fn seal_l1_batch(
+    storage: &mut StorageProcessor<'_>,
+    number: L1BatchNumber,
+) -> anyhow::Result<()> {
+    let header = L1BatchHeader::new(
+        number,
+        number.0.into(),
+        Address::repeat_byte(1),
+        BaseSystemContractsHashes::default(),
+        ProtocolVersionId::latest(),
+    );
+    storage
+        .blocks_dal()
+        .insert_l1_batch(&header, &[], BlockGasCount::default(), &[], &[])
+        .await?;
+    storage
+        .blocks_dal()
+        .mark_miniblocks_as_executed_in_l1_batch(number)
+        .await?;
+    let metadata = create_l1_batch_metadata(number.0);
+    storage
+        .blocks_dal()
+        .save_l1_batch_metadata(number, &metadata, H256::zero(), false)
+        .await?;
+    Ok(())
+}
+
+#[derive(Debug)]
+struct SnapshotBasics {
```

Comment:
What name would you prefer? It's in a test module and implements `HttpTest` crate which IMO describes its purpose just fine. I'm not a fan of enterprise-y `SnapshotBasicsHttpTest` naming schema, but that's not a strong preference – if you think it'll convey the intention better, I'll switch it.

Reply:
I didn't have any specific name in my mind, but I think that using name `SnapshotBasicsHttpTest` or similar to it would have made it a lot easier for me to understand what this does if this is the first time I see a HttpTest